inventory/by_cloud/aws

```diff
@@ -1,11 +1,16 @@
 [lib_ops_staging]
 lib-ops-staging ansible_host=34.233.30.106
+[pdc_globus]
+pdc-globus-deposit ansible_host=44.217.203.68
+pdc-globus-ops_pul_globus ansible_host=34.233.30.106
```

Comment:
@acozine Was this a mistaken build on ops part? I'm thinking maybe it should either be deleted or in the lib_ops gorup
```suggestion
```